teacher_class.rb

```diff
@@ -0,0 +1,13 @@
+require './Person_Class'
+
+# This is the teacher class inherit from person
+class Teacher < Person
+  def initialize(specialization)
```

Comment:
- [ ] [REQUIRED] Please give correct value to in line 5, it would like this ``def initialize(age, specialization, name = "Unknown", parent_permission = true)``